src/CattyTests/PlayerEngine/Sensors/Date/DateWeekdaySensorTest.swift

```diff
@@ -90,25 +90,25 @@ final class DateWeekdaySensorTest: XCTestCase {
 
     func testConvertToStandardized() {
         /* test Sunday */
-        XCTAssertEqual(7, Int(sensor.convertToStandardized(rawValue: 1)))
+        XCTAssertEqual(7, Int(sensor.convertToStandardized(rawValue: 1, landscapeMode: false)))
 
         /* test Monday */
-        XCTAssertEqual(1, Int(sensor.convertToStandardized(rawValue: 2)))
+        XCTAssertEqual(1, Int(sensor.convertToStandardized(rawValue: 2, landscapeMode: false)))
 
         /* test Tuesday */
-        XCTAssertEqual(2, Int(sensor.convertToStandardized(rawValue: 3)))
+        XCTAssertEqual(2, Int(sensor.convertToStandardized(rawValue: 3, landscapeMode: false)))
 
         /* test Wednesday */
-        XCTAssertEqual(3, Int(sensor.convertToStandardized(rawValue: 4)))
+        XCTAssertEqual(3, Int(sensor.convertToStandardized(rawValue: 4, landscapeMode: false)))
 
         /* test Thursday */
-        XCTAssertEqual(4, Int(sensor.convertToStandardized(rawValue: 5)))
+        XCTAssertEqual(4, Int(sensor.convertToStandardized(rawValue: 5, landscapeMode: false)))
 
         /* test Friday */
-        XCTAssertEqual(5, Int(sensor.convertToStandardized(rawValue: 6)))
+        XCTAssertEqual(5, Int(sensor.convertToStandardized(rawValue: 6, landscapeMode: false)))
 
         /* test Saturday */
-        XCTAssertEqual(6, Int(sensor.convertToStandardized(rawValue: 7)))
+        XCTAssertEqual(6, Int(sensor.convertToStandardized(rawValue: 7, landscapeMode: false)))
```

Comment:
Please also add a test for `convertToStandardized` when called with `landscapeMode` `true`.